src/lib/components/labeledInput/labeledInput.module.scss

```diff
@@ -79,6 +79,13 @@
     }
   }
 
+  input[readonly], input[readonly]:focus {
```

Comment:
Is it working? I think you need `@include themify() {` because you need to run *themed* inside `themefy` block.

Reply:
Sorry! That was sloppy! 💩
Fixed it. (I think it worked accidentally.)